qiskit/schemas/backend_configuration_schema.json

```diff
@@ -29,7 +29,8 @@
           },
         "coreconfig": {
             "type": "object",
-            "required": ["backend_name", "backend_version", "n_qubits", "basis_gates", "gates", "local", "simulator", "conditional", "memory", "max_shots"],
+            "required": ["backend_name", "backend_version", "n_qubits", "basis_gates", "gates", "local", "simulator", "conditional", "memory", "max_shots",
+                         "supported_instructions"],
```

Comment:
This is a breaking change and would be a major version bump. This should not be required.

Reply:
Fair pt. Addressed in https://github.com/Qiskit/qiskit-terra/pull/5118/commits/6024d0ebf15b3be6721761abcf3468a61560d40b.